config/crd/patches/conversion.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patches make Kubernetes just replace the version, and
+# assume the same schema, for CRD conversion from v1alpha2 to v1beta1.
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: imagerepositories.image.toolkit.fluxcd.io
+spec:
+  conversion:
+    strategy: None
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: imagepolicies.image.toolkit.fluxcd.io
+spec:
+  conversion:
+    strategy: None
```

Comment:
`None` is the default conversion strategy, no need for this patch, worked fine without it when we promoted v1alpha1 to v1alpha2.

Reply:
OK, if there's no need to be explicit about it, I can remove it -- fewer files innit :-)